libcontainer/cgroups/systemd/v1.go

```diff
@@ -394,6 +418,13 @@ func (m *legacyManager) Set(container *configs.Config) error {
 		return err
 	}
 
+	// Unfreeze the cgroup.
```

Comment:
Seems covered in `defer m.Freeze(configs.Thawed)`

Reply:
I wanted to make sure we unfroze the cgroup before we ran the fsManager (to limit the length the time the container remains frozen, as well as to avoid stepping on our toes with the fsManager also writing to the freezer cgroup).

However, thinking about this more the `defer` is incorrect (as is using `configs.Thawed` directly) because if you're dealing with a user-frozen container (or the user requested we create a frozen container) then we won't do what they're expecting. I'll fix that now.